packages/appstore/src/components/transaction-list/transaction-list.tsx

```diff
@@ -280,10 +313,8 @@ const FiatTransactionList = () => {
                         let icon_type = 'fiat';
                         let is_deriv_apps = false;
                         if (transaction.action_type === 'transfer') {
```

Comment:
@rostislav-deriv There is a lot going on in this `map` function in the middle of the JSX, Can we extract it into a separate component?

Reply:
Moved it to `useWalletTransactions`